src/components/services/Mealie.vue

```diff
@@ -6,7 +6,9 @@
         <template v-if="item.subtitle">
           {{ item.subtitle }}
         </template>
-        <template v-else-if="meal"> Today: {{ meal[0].recipe.name }} </template>
+        <template v-else-if="meal">
+          Today: {{ (meal && meal.length > 0) ? meal[0].recipe.name : 'Nothing planned' }}
```

Comment:
```suggestion
          Today: {{ meal.length > 0 ? meal[0].recipe.name : 'Nothing planned' }}
```
Because it's already checked on line 9 ?  
Maybe better: Use a computed property to put the logic away from the template?